components/Typography/tests/unit/MaterialScalableFontTests.m

```diff
@@ -0,0 +1,266 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please rename this class to match the class being tested. Most likely something like, `UIFont_MaterialScalableTests.m`. It seems you named the interface correctly, it's just the file name that's mismatched.